pdns/dbdnsseckeeper.cc

```diff
@@ -589,6 +589,26 @@ bool DNSSECKeeper::getTSIGForAccess(const DNSName& zone, const string& master, D
   return false;
 }
 
+bool DNSSECKeeper::unSecureZone(const DNSName& zone, string& error, string& info) {
```

Comment:
`info` doesn't seem to be used?

Reply:
Correct, should we ever add more functionality to this function, then it might be. I kept is similar to `rectifyZone()`.